sdk/ai/azure-ai-projects/samples/agents/sample_agents_code_interpreter_attachment_enterprise_search.py

```diff
@@ -62,7 +60,12 @@
     ds = VectorStoreDataSource(asset_identifier=asset_uri, asset_type=VectorStoreDataSourceAssetType.URI_ASSET)
 
     # create a message with the attachment
-    attachment = MessageAttachment(data_sources=[ds], tools=code_interpreter.definitions)
```

Comment:
Would the earlier way also work if you have

    code_interpreter = CodeInterpreterTool()

    agent = project_client.agents.create_agent(
        model="gpt-4-1106-preview",
        name="my-assistant",
        instructions="You are helpful assistant",
        tools=code_interpreter.definitions,
    )

 attachment = MessageAttachment(data_source=ds, tools=code_interpreter.definitions)

Reply:
Fixed.